src/MultiplicationTable.ts

```diff
@@ -1,5 +1,41 @@
 export class MultiplicationTable {
   public render(start: number, end: number): string{
-    return '1*1=1'
+    const isNumberValid = this.isNumberValid(start, end)
+    if (isNumberValid) {
```

Comment:
maybe it's not necessary to use a variable to store this bool
"if (this.isNumberValid(start, end))" is ok